### PR TITLE
Sync parse functions

### PR DESCRIPTION
**Pull request by seka**
Expose functions `parseSync` and `parseFromFilesSync`.

And added those test.

### CODE REVIEW

File: test/.editorconfig [hunk 1]
```diff
@@ -0,0 +1,15 @@
+[*]
```
Comment (jedmao) [inline test/.editorconfig]: The root `.editorconfig` file already has these settings.

File: test/.editorconfig [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.py]
```
Comment (jedmao) [inline test/.editorconfig]: How did Python enter the mix?

File: test/.editorconfig [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+indent_style = space
+indent_size = 4
+
+[*.js]
+indent_style = tab
```
Comment (jedmao) [inline test/.editorconfig]: Why?

File: test/.editorconfig [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+[lib/**.js]
+indent_style = mixed
```
Comment (jedmao) [inline test/.editorconfig]: Why mixed?

**Comment by seka** [comment]
thank you for review.
It was wrong to added a `test/.editorconfig` for a test.
I retouch it to use an existing  `.editorconfig`.

**Comment by jedmao** [comment]
Can you add some readme docs for these new methods?

**Comment by seka** [comment]
sorry... I am not confident that I will explain that well.

**Comment by jedmao** [comment]
If you're worried about the English, I can correct the spelling/grammar for you, but this really needs some readme info and I think you're the best to add it.

**Comment by seka** [comment]
ok! I'll try it.

**Comment by seka** [comment]
@jedmao 
I was waiting for you. What happened?

**Comment by jedmao** [comment]
Sorry @seka ! Must have missed the notification. It's merged in and published as 0.13.0. Thanks!

**Pull request merged**